template.yaml

```diff
@@ -52,6 +52,9 @@ Parameters:
     Type: CommaDelimitedList
   NetworkName:
     Type: String
+  LockfileDelay:
+    Type: Number
+    Default: 600
```

Comment:
this is broken out as a parameter as a dev convenience; non-dev environments won't futz with this